tools/terraform_sync_tool/modules/bq-setup/variables.tf

```diff
@@ -0,0 +1,113 @@
+variable "description" {
```

Comment:
Remove the variables you don't use

Reply:
Removed [Rename module to bigquery & provide default datasetID](https://github.com/GoogleCloudPlatform/bigquery-utils/pull/290/commits/e85bd2110e5113d32d5d4e3cc3160be6e37527c9)